components/tools/OmeroPy/test/integration/clitest/test_import.py

```diff
@@ -287,7 +286,7 @@ def testDebugArgument(self, tmpdir, capfd, level, prefix):
         self.cli.invoke(self.args, strict=True)
         o, e = capfd.readouterr()
         levels = self.parse_debug_levels(o)
-        assert set(levels) <= set(debug_levels[debug_levels.index(level):])
+        assert set(levels) <= set(debug_levels[debug_levels.index(level):]), o
```

Comment:
It does look odd though! maybe if `o, e` were `out, err` or `stdout, stderr` it might be clearer?